tests/e2e/test_create_project.py

```diff
@@ -0,0 +1,29 @@
+from playwright.sync_api import Page, expect
+
+
+def test_e2e_create_project(page: Page):
```

Comment:
```suggestion
def test_create_project(page: Page):
```

Or otherwise maybe prefix all e2e tests with `e2e`?

Reply:
Renamed all with e2e